src/Scripty.CustomTool/BaseCodeGenerator.cs

```diff
@@ -1,4 +1,6 @@
-﻿
+﻿//Dave Glick - is this just template copyright stuff that was brought in or 
+// is this something else?
+
```

Comment:
Yeah, the two files with this copyright header were copied wholesale from a Microsoft example.